### ISSUE TITLE
ignore_paths

### ISSUE THREAD

**Issue by braindev** [tockins/realize#9]
I am trying to use the `ignore_paths` configuration to ignore a large `node_modules` directory.  It doesn't seem to be working.  I can see that the path walking is going down into the `node_modules` directory.  Am I doing something wrong?

When I run `realize run` I get an error: `open /dev/null: too many open files`

Here's the config file:

```yaml
version: v1.0
projects:
- app_name: whiz
  app_path: /
  app_run: true
  app_bin: true
  app_fmt: false
  app_watcher:
    preview: true
    paths:
    - /
    ignore_paths:
    - node_modules
    - vendor
    - bin
    exts:
    - .go
```

**Comment by conventi** [comment]
@braindev this is a known bug. We will fix as soon as possibile (tomorrow). See issue #5 and pull request #7.

**Comment by conventi** [comment]
@braindev I've merged the pull request #7. Now ignore_paths should work.
